README.md

```diff
@@ -22,10 +22,13 @@ jobs:
     - uses: bufbuild/buf-push-action@v1
       with:
         buf_token: ${{ secrets.BUF_TOKEN }}
+        enable_draft: true
 ```
 
-With this configuration, the `buf` CLI pushes the [configured module][buf-yaml] to the BSR upon
-merge using a Buf API token to authenticate with the [Buf Schema Registry][bsr] (BSR).
+With this configuration, the `buf` CLI pushes the [configured module][buf-yaml] to the BSR using 
+a Buf API token to authenticate with the [Buf Schema Registry][bsr] (BSR), upon a pull request 
+[opened, synchronize, or reopened][github-workflow], and will push as a draft when the triggering 
+branch is not `main`.
```

Comment:
```suggestion
With this configuration, upon a pull request [opened, synchronize, or reopened][github-workflow] 
the `buf` CLI pushes the [configured module][buf-yaml] to the BSR using the provided to 
authenticate the request. When the triggering branch is not `main`, the commit will be pushed 
as a [draft][<put-draft-docs-link-here>].
```

Some run on sentences here, I think this captures it? Still needs a link to the draft docs.

Reply:
updated